ui/selectors/selectors.js

```diff
@@ -175,10 +175,6 @@ export function getCurrentKeyring(state) {
   return keyring;
 }
 
-export function isEIP1559Account() {
-  return true;
-}
```

Comment:
Lol what. Guess when the last hardware wallet moved to support EIP1559 this was done. Make a lot of sense to remove!